hamilton/io/materialization.py

```diff
@@ -26,7 +26,7 @@ def __new__(cls, name, bases, clsdict):
         clsobj.__annotations__ = {}
         return clsobj
 
-    def __getattr__(cls, item: str) -> "MaterializerFactory":
```

Comment:
We need to move these classes around too -- I can do that if you're slammed

Reply:
Hmm I think we don't -- this is now delayed in evaluation. So long as this doesn't break/complain if you put `from __future__ import annotations` then it should be fine!